app/jobs/alma_full_dump_transfer_job.rb

```diff
@@ -0,0 +1,76 @@
+require 'net/sftp'
+
+class AlmaFullDumpTransferJob < ActiveJob::Base
+  queue_as :default
+
+  REMOTE_BASE_PATH = "/home/alma".freeze
+
+  def perform(dump:, job_id:, type_constant: 'BIB_RECORDS')
```

Comment:
Question: Why is `type_constant:` an argument? Is there a case where it might be something else for a full dump transfer job?